bindata/network/ovn-kubernetes-ipsec/ovn-ipsec.yaml

```diff
@@ -183,19 +183,16 @@ spec:
           fi
           echo "ovnkube-node has configured node."
 
+          # After a restart of this container (or on initial startup), we flush xfrm state and policy
+          # before we start pluto and ovs-monitor-ipsec in order to start in a known good state. This
+          # will result in a small interruption in traffic until pluto and ovs-monitor-ipsec start again.
+          ip x s flush
+          ip x p flush
```

Comment:
Is there any chance this might flush someone else's state? (ie, is there any plausible way some other operator might be doing something with "ip x"? Or is that command basically limited to IPsec stuff only, and so there's no way someone else could be using it without interfering with us anyway?)

Reply:
@danwinship - It is namespaced so it will only affect the default namespace (host namespace). Therefore, we should be the only ones modifying this.